src/en/space-station-14/roleplay-lore/proposals/Departmental-Operating-Procedure.md

```diff
@@ -0,0 +1,122 @@
+# Departmental Operating Procedure (DOP)
+
+## Abstract
+
+--- 
+
+The DOP is an in-character set of guidelines oriented towards MRP servers. Each department will have it's own 'Operating Procedure' that will prepare it for any encounters it may face while in space. This will serve as a way to encourage roleplay and more belivable in-character behavior. 
+
+The DOP will act as a guideline for departments to follow but not rules. This will allow for chaos to still ensue even when everything goes according to plan. 
+
+## Core Idea
+
+My intention with a DOP is to encourage Roleplay but not to enforce it with server rules, this instead is a set of in-character books and guidebook entries that will push players into a more RP centric enviorment. This proposal is more focused towards MRP servers but these guidelines can still be found and used within LRP servers. 
```

Comment:
According to this, DOP is basically a flavor document without anything being enforced. Isn't this just an attempt in official lore of sorts?

Reply:
It largely is just a lore thing at this point. I would like for it to be more like the original sop in terms of needing to be followed but I am doubtful that would be approved.